compiler/src/iree/compiler/Dialect/Stream/Transforms/SpecializeEncodings.cpp

```diff
@@ -138,6 +138,32 @@ updateBindingEncodings(FunctionOpInterface funcOp,
   return success();
 }
 
+/// Returns the operands encodings and result encodings from the `dispatchOp`.
```

Comment:
Could you explain what the order of the returned encoding attrs is and if it matters?